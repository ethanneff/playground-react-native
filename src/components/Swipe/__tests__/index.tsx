import React from 'react';
import { create } from 'react-test-renderer';
import { Swipe } from '..';

describe('swipe', () => {
  it('renders correctly', () => {
    expect.hasAssertions();
    const tree = create(<Swipe />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
