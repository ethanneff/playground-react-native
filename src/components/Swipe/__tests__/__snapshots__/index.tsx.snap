// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`swipe renders correctly 1`] = `
<View
  style={
    Array [
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
/>
`;
